claims_test.go

```diff
@@ -1,39 +1,56 @@
 package jwtauth
 
 import (
+	"encoding/json"
 	"testing"
 
 	"github.com/go-test/deep"
 )
 
 func TestGetClaim(t *testing.T) {
+	data := `
+{
```

Comment:
Not 100% sure but I _think_ it'll still parse if you tab it over to be at the nested level for data.

Reply:
👍